torchvision/prototype/models/efficientnet.py

```diff
@@ -94,7 +94,17 @@ class EfficientNet_B1_Weights(WeightsEnum):
             "acc@5": 94.186,
         },
     )
-    default = ImageNet1K_V1
+    ImageNet1K_V2 = Weights(
+        url="https://download.pytorch.org/models/efficientnet_b1-c2e40cbf.pth",
+        transforms=partial(ImageNetEval, crop_size=224, resize_size=232, interpolation=InterpolationMode.BILINEAR),
+        meta={
+            **_COMMON_META,
```

Comment:
You should have also defined `"size": (240, 240),` in the meta. This indicates the size of the image that goes in the network.

You also must overwrite the `interpolation` value on the meta as unlike all other models, you use bilinear.